translations/da.po

```diff
@@ -48241,3 +48241,28 @@ msgstr "Log ud"
 #: /system/ajax
 msgid "Authenticate"
 msgstr "<p>Du har indtastet et forkert lånernummer/cpr eller en forkert pinkode, eller du er ikke oprettet som låner ved biblioteket.</p><p>Har du glemt din pinkode så kontakt dit lokale  <a href="@url">bibliotek</a>.</p>"
+
+#: ding_item_cache.module:35
+msgid "Select those modules which cache has to be cleared"
+msgstr "Vælg hvilke moduler der skal renses for mellemlagring"
+
+#: ding_item_cache.module:42
+msgid "Clear caches"
+msgstr "Ryd mellemlagring"
+
+
+#: ding_item_cache.module:57
+msgid "Ding item cache has been cleared"
+msgstr "Materialekarusellen er blevet renset"
```

Comment:
"Materialekarusellen er blevet renset" -> "Mellemlagringen for materialer er blevet renset"